isso/js/app/templates/comment-loader.js

```diff
@@ -0,0 +1,9 @@
+var html = function (globals) {
+  // De-structure globals dict
+  ({comment, pluralize} = {...globals});
```

Comment:
Do you know any cleaner way to extract only `comment` and `pluralize` from the `globals` object, @Kerumen ? I only settled on using destructuring because _some guy on stackoverflow_ said to use it.

Reply:
You can simply do:

```js
var { comment, pluralize } = globals;
```

On a side note, as you are now using destructuring and perhaps template literals, you could switch `var` to `const` / `let`, no? Let's write modern JS! 😄

Reply:
I can understand your enthusiasm, but I'm also wary of breaking Isso for a lot of people who are stuck on old browsers (or their assistive technologies don't understand ES6 yet, or they're on locked-down ancient corporate machines, etc.).

Is transpiling to older JS version a thing that can be relied upon? Such that we might emit an ES5-compatible `embed.legacy.js` through `babel`?